community/kernel/src/main/java/org/neo4j/graphdb/factory/GraphDatabaseSettings.java

```diff
@@ -838,6 +838,13 @@ public String providerName()
     public static final Setting<String> default_advertised_address =
             setting( "dbms.connectors.default_advertised_address", STRING, "localhost" );
 
+    @Description( "Default precision of temporal time based values. Corresponds to the number of value " +
+            "figures in the nanosecond part, i.e. 0 corresponds to second , 3 to millisecond, 6 to " +
+            "microsecond and 9 to nanosecond precision. Must be an integer in the range 0 - 9." )
```

Comment:
I wonder if it is worth adding the windows example of 7 decimal points here since that is a valid (perhaps even likely) option for the .NET driver?

Reply:
I guess doing that will also emphasise the key reason for this feature. 6 is used by default because it is supported by all client libraries. 7 for windows .NET, and 9 for Java.